pynest/nest/raster_plot.py

```diff
@@ -106,21 +101,54 @@ def from_file(fname, **kwargs):
 
     Parameters
     ----------
-    fname : str
-        Name of file
+    fname : str or tuple(str) or list(str)
+        File name or list of file names
     kwargs:
         Parameters passed to _make_plot
     """
-
-    if nest.is_iterable(fname):
-        data = None
-        for f in fname:
-            if data is None:
-                data = numpy.loadtxt(f)
-            else:
-                data = numpy.concatenate((data, numpy.loadtxt(f)))
+    if isinstance(fname, str):
+        fname = [fname]
+
+    if isinstance(fname, (list, tuple)):
+        try:
+            global pandas
+            pandas = __import__('pandas')
+            from_file_pandas(fname, **kwargs)
+        except ImportError:
+            from_file_numpy(fname, **kwargs)
     else:
-        data = numpy.loadtxt(fname)
+        print >> sys.stderr, 'fname should be of str/list(str)/tuple(str).'
```

Comment:
Didn't now this redirection was possible, in Python. But I am rather sure this line won't work in Python 3, or?